src/app/globals.css

```
@@ -1,3 +1,65 @@
+  .dark {
+    --background: 10 10 10;
+    --foreground: 249 250 251;
+    --muted: 156 163 175;
+    --border: 31 31 31;
+    --surface: 17 17 17;
+    --brand: 243 111 54;
+  }
+
+  html {
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.10.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)